issue_maker.py

```diff
@@ -293,14 +295,22 @@ def update():
             with open('unsupported.txt', 'a') as f:
                 f.write('{song} by {artist}\n'.format(song=song, artist=artist))
 
-            issue = create_issue(song, artist, version, stripped)
-            if issue['status_code'] == 201:
-                print('Created issue on the GitHub repo for {song} by {artist}.'.format(song=song, artist=artist))
-                return 'Lyrics for that song may not exist on Genius. Created issue on the GitHub repo for {song} by ' \
-                       '{artist} to investigate further. \n{link}'.format(
-                        song=song, artist=artist, link=issue['link'])
+            if re.fullmatch(asrg, song) and re.fullmatch(asrg, artist):
```

Comment:
Can you use an f-string? The backend runs on Python 3.8 so it shouldn't be a problem

Reply:
Sure!

Reply:
You mean use f-string in whole file or only in this line specifically?

Reply:
We want to do it for the whole file eventually but since you wrote just this line, that's fine too :)

Reply:
I replaced not all but most of them for f-strings.